src/reporters/errors/templates.js

```diff
@@ -7,6 +7,16 @@ function escapeNewLines (str) {
     return str.replace(/(\r\n|\n|\r)/gm, '\\n');
 }
 
+function getStepCode (str) {
+    var strArr = str.match(/[^\r\n]+/g);
+    var code   = '';
+
+    while (strArr.length > 1)
+        code += `<code-line>${strArr.shift()}</code-line>`;
+
+    return code + `<last-code-line>${strArr.pop()}</last-code-line>`;
```

Comment:
Why not `split` here? BTW, we shouldn't replace multiple new lines with just single new line to preserve original formatting, so we need to change RegExp, In general it can be rewritten as:

```js
function getStepCode (str) {
    var lines = str.split(/\r\n?/g);
    var last  = lines.length - 1;

    return lines
        .map((line, idx) => {
            return idx === last ? `<last-code-line>${line}</last-code-line>` : `<code-line>${line}</code-line>`
        })
        .join('');
}
```